Sources/TestSupportExecutable/main.swift

```diff
@@ -0,0 +1,56 @@
+import Basic
+import POSIX
+import libc
```

Comment:
I forgot to comment here, I think the new parser will automatically solve this and generate usage text. We then probably only need minimal comments here, I already have a FIXME for moving to new parser once we have it